OpenOversight/app/main/model_view.py

```diff
@@ -127,10 +128,20 @@ def edit(self, obj_id, form=None):
             elif hasattr(form, "created_by"):
                 form.created_by.data = current_user.get_id()
 
+            # TODO: Add created_at field to forms
+            if (
+                hasattr(obj, "created_at")
+                and hasattr(form, "created_at")
+                and getattr(obj, "created_at")
+            ):
+                form.created_at.data = obj.created_at
+            elif hasattr(form, "created_at"):
+                form.created_at.data = now
```

Comment:
The last merged PR removed the `created_by` fields from the forms, and I think it'd make sense if we didn't have the `created_at` fields on there either. The field should be staying the same in the DB as long as we don't add update the `created_at` field, yeah?